bin/next

```diff
@@ -5,6 +5,24 @@ import { spawn } from 'cross-spawn'
 import { watchFile } from 'fs'
 import pkg from '../../package.json'
 
+if (pkg.peerDependencies) {
+  let peerDependencyUnavailable = false
+
+  Object.keys(pkg.peerDependencies).forEach(dependency => {
+    try {
+      // When 'npm link' is used it checks the clone location. Not the project.
+      require.resolve(dependency)
+    } catch (err) {
+      console.log(`${dependency} not found. Please install ${dependency} using 'npm install ${dependency}'`)
+      peerDependencyUnavailable = true
+    }
+  })
+
+  if (peerDependencyUnavailable) {
```

Comment:
eventually a dumb question, but do we actually need react in every case? what if we're using preact or inferno? <- never used them, but saw it in the example folder... do we still need react installed?

If we don't strictly require them we shouldn't kill the application here.